Add IPVanish and NordVPN

spazvt opened the pull request:
add IPVanish and NordVPN service provider, originally added by @deadr0ck to fork
https://github.com/acceptbitcoincash/acceptbitcoincash

spazvt committed:
add IPVanish and NordVPN from 'acceptbitcoincash'

add IPVanish and NordVPN service provider, originally added by
@deadr0ck to fork
https://github.com/acceptbitcoincash/acceptbitcoincash

SonnevilleJ commented on _data/security.yml:
```diff
@@ -90,6 +90,24 @@ websites:
+      software: Yes
+      twitter: nordvpn
+      facebook: NordVPN
+      doc: https://nordvpn.com/blog/secure-accounts-two-factor-authentication/
```
This linked doc does not include instructions to enable two-factor authentication for NordVPN.

Carlgo11 replied:
I can confirm that NordVPN doesn't offer two factor authentication to it's users.

spazvt committed:
NordVPN doesn't support 2FA. rm misleading doc.

* mark NordVPN as not supporting 2FA, per @carlgo11.
* remove doc link because it only described how to use 2FA on
OTHER sites.

spazvt committed:
IPVanish doesn't support 2FA.

Logged into IPVanish, I can't find any evidence of an ability to use
2FA. I searched their help forum and FAQ and found no evidence either.

mxxcon reviewed (changes requested):
Please use Nord logo from their website https://nordvpn.com/wp-content/themes/nordvpn-bootstrap/build/1524660327/images/favicons/favicon-32x32.png

spazvt committed:
change NordVPN logo to non-text version

NordVPN logo from:
https://nordvpn.com/wp-content/themes/nordvpn-bootstrap/build/1524660327/images/favicons/favicon-32x32.png

spazvt committed:
Merge branch 'master' into addIPVanishAndNordVPN

Carlgo11 commented on _data/security.yml:
```diff
@@ -105,12 +105,26 @@ websites:
+      url: https://www.ipvanish.com
+      img: ipvanish.png
+      tfa: No
+      twitter: ipvanish
```
Please change the Twitter tag to `IPVanish`.

Carlgo11 commented on _data/security.yml:
```diff
@@ -105,12 +105,26 @@ websites:
+      url: https://www.nordvpn.com
+      img: nordvpn.png
+      tfa: No
+      twitter: nordvpn
```
Please change the Twitter handle to `NordVPN`.

Carlgo11 commented on _data/security.yml:
```diff
@@ -105,12 +105,26 @@ websites:
       tfa: No
 
+    - name: NordVPN
+      url: https://www.nordvpn.com
```
Change the url to `https://nordvpn.com/`

spazvt committed:
shorten URL https://nordvpn.com

* Shorten URL https://nordvpn.com
* Capitalize IPVanish twitter handle.
* Capitalize NordVPN twitter handle.

Carlgo11 reviewed (approved):
Looks good

jamcat22 reviewed (approved):
Thanks for the PR @spazvt!
